### ISSUE TITLE
main file missing

### ISSUE THREAD

**Issue by Gaoyuan-Liu-vub** [ViktorLaurens/MMMP#1]
Hey,

I can't find the `main.py` file in the repo. Can you specify which file I should run to see the performance?

Thanks!

**Comment by ViktorLaurens** [comment]
Yes you can run the files in the simulations, tests and experiments directories to run algorithms, for instance: python -m sims.cbs_prm

I will add a main file this afternoon!

Vriendelijke groeten,
Viktor
________________________________
From: Gaoyuan-Liu-vub ***@***.***>
Sent: Monday, August 19, 2024 12:55:49 PM
To: ViktorLaurens/MMMP ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ViktorLaurens/MMMP] main file missing (Issue #1)


Hey,

I can't find the main.py file in the repo. Can you specify which file I should run to see the performance?

Thanks!

**Comment by ViktorLaurens** [comment]
So I added a main file for you to run.

Kind regards,
Viktor

Vriendelijke groeten,
Viktor
________________________________

**Comment by Gaoyuan-Liu-vub** [comment]
I successfully ran the first demo.
Good job!
